### ISSUE TITLE
Read The Docs version switcher is not reachable using only keyboard (tab key)

### ISSUE THREAD

**Issue by gabalafou** [Quansight-Labs/czi-scientific-python-mgmt#86]
```[tasklist]
### Tasks
- [ ] File issue with Read the Docs
```

**Comment by gabalafou** [comment]
Path forward: open issue on Read The Docs

**Comment by trallard** [comment]
@humitos 👋  can you remind me where we should log RTD-related accessibility issues

**Comment by humitos** [comment]
👋🏼 , it depends 😅 

* Sphinx theme: https://github.com/readthedocs/sphinx_rtd_theme/
* Addons: https://github.com/readthedocs/addons
* Dashboard: https://github.com/readthedocs/ext-theme

Since this issue talks about the version selector, it should probably go on the addons or Sphinx theme, depending if you the issue is about the version selector that's floating bottom right (addons) or integrated into the navigation bar at bottom left (sphinx theme).

I hope this directions help 😄

**Comment by trallard** [comment]
Thanks for the quick response @humitos seems the right place would be the add ons, same as #87 

cc/ @gabalafou

**Comment by trallard** [comment]
Blocked until we check what is needed to update for the new RTD assets (will no longer be injected in the built docs)
